packages/applications-service-api/__tests__/setup-jest.js

```diff
@@ -6,3 +6,4 @@ process.env.MYSQL_DATABASE = 'database';
 process.env.MYSQL_HOST = 'host';
 process.env.MYSQL_PORT = '3306';
 process.env.APPLICATIONS_WEB_BASE_URL = 'http://forms-web-app:9004';
+process.env.FEATURE_ALLOW_WELSH_TRANSLATION = 'true';
```

Comment:
If you could use the config.js setup, then the config export can be mocked instead of adding it here.